store/sqlstore/channel_store_categories.go

```diff
@@ -674,10 +674,7 @@ func (s SqlChannelStore) UpdateSidebarCategories(userId, teamId string, categori
 				Delete("SidebarChannels").
 				Where(
 					sq.And{
-						sq.Or{
-							sq.Eq{"ChannelId": originalCategory.Channels},
-							sq.Eq{"ChannelId": updatedCategory.Channels},
-						},
+						sq.Eq{"ChannelId": srcCategory.Channels},
```

Comment:
Just in case we need to come back to this later, this is the major change here. The rest is just renaming